src/evict/evict_lru.c

```diff
@@ -785,7 +788,7 @@ __evict_pass(WT_SESSION_IMPL *session)
         if (!__evict_update_work(session))
             break;
 
-        __wt_verbose_debug2(session, WT_VERB_EVICTION,
+        __wt_verbose_info(session, WT_VERB_EVICTION,
```

Comment:
The WT-13383 description says that the aim is to reduce noise in logs.
Some of these changes increase the amount of logging on lower levels.
While thread creation or destruction is rare, I understand why it's good to pull it to lower levels.
Eviction passes however can happen quite rapidly - 100s times per second. Is it really useful pulling thi 2 levels up?

Reply:
Yes, I have that concern that these logs can be too many. But I want to provide some information about eviction activity with default logging, otherwise, these logs will rarely enabled in the production to get more information.

I will run a MongoDB patch build to confirm how frequently these logs are getting logged to change their log levels.

Reply:
Maybe it's worth adding FTDC counters for these?

Reply:
The FTDC counters don't provide the details of dhandle based on which these counters are getting updated.

Reply:
After testing these logs are getting generated too many and I am converting them into `debug1` level.